test/test_combine_mul_sum.cpp

```diff
@@ -187,4 +187,44 @@ TEST_F(CombineMulSumAsMmaTest, AmpereMulSumToMatmul_Schedule) {
   }
 }
 
+TEST_F(CombineMulSumAsMmaTest, UseMatmulScheduler) {
+  // Keep multiples of 8 to keep vectorizable.
+  int M = 504, N = 136, K = 248;
+  const auto layout = MmaLayout::TT;
+
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  auto tv0 = makeContigTensor(2, DataType::Half);
+  auto tv1 = makeContigTensor(2, DataType::Half);
+
+  fusion->addInput(tv0);
+  fusion->addInput(tv1);
+
+  auto tv2 = matmul(tv0, tv1, layout, true, true);
+  fusion->addOutput(tv2);
+  ASSERT_TRUE(ir_utils::getOpsOfType<MmaOp>(fusion.get()).empty());
+
+  auto t0 = matmulAtInput(layout, TensorMatmulPos::A, at::kHalf, M, N, K);
+  auto t1 = matmulAtInput(layout, TensorMatmulPos::B, at::kHalf, M, N, K);
+  auto tref = atMatmul(t0, t1, layout);
+
+  FusionExecutorCache executor_cache(std::move(fusion));
+  auto outputs = executor_cache.runFusionWithInputs({t0, t1});
+  // Ensure there's a mma op.
+  ASSERT_FALSE(
+      ir_utils::getOpsOfType<MmaOp>(executor_cache.getMostRecentKernelRuntime()
+                                        ->executors()
+                                        .at(0)
+                                        .kernel())
+          .empty());
+
+  NVF_CHECK(
```

Comment:
nit: use `EXPECT_FALSE` for these two checks.

Reply:
`EXPECT_FALSE` in particular, as opposed to `ASSERT_FALSE` or `NVF_CHECK`, will allow the test to proceed, but will give an error message and cause the test to fail after it runs to completion. This is more informative for conditions that are not guarding downstream checks. For example, you'd use `ASSERT_*` to check the size of a vector if you are going to extract an element from that vector later. Since nothing else depends on this condition, I think it's better as `EXPECT`. Note that of course this is not very important since it doesn't affect whether or not the test fails :-D

Reply:
Changed it to `EXPECT_FALSE` -- but I assert when checking for a mma op.